config.yaml

```diff
@@ -20,10 +20,10 @@ sources:
   - agora_proteomics:
     agora_proteomics_files: &agora_proteomics_files
       - name: proteomics
-        id: syn18689335.3
+        id: syn18689335.4
```

Comment:
Do we know what's changed in this new version?

Reply:
I believe you said they were identical but we should bump the version so we don't wonder why we're not using the latest version later. I just checked and the two versions have the same md5 sum on Synapse so they are definitely identical.

Reply:
I'm glad you remember the things I say... 😊